apps/docs/src/app/core/component-docs/menu/menu-docs.component.html

```diff
@@ -22,37 +34,68 @@
-    Menu with Group
+<fd-docs-section-title [id]="'optionalSeparator'" [componentName]="'menu'">
+    Menu with separator and compact mode
 </fd-docs-section-title>
+<description>
+    To add visual separation line between menu items use <code>fd-menu-separator</code> component.<br/>
+    To display Menu in compact mode use <code>[compact]="true"</code>.
+</description>
 <component-example>
-    <fd-menu-group-example></fd-menu-group-example>
+    <fd-menu-separator-example></fd-menu-separator-example>
 </component-example>
-<code-example [exampleFiles]="menuGroup"></code-example>
+<code-example [exampleFiles]="menuSeparator"></code-example>
 
 <separator></separator>
 
-<fd-docs-section-title [id]="'optionalSeparator'" [componentName]="'menu'">
-    Menu with Optional Separator
+<fd-docs-section-title [id]="'menu-with-submenu'" [componentName]="'menu'">
+    Menu with submenu
 </fd-docs-section-title>
-<description>Each menu item can have an optional separator shown as a line beneath the item.</description>
+<description>
+    Each Menu item can have its submenu. To create submenu:
+    <ul>
+        <li>create an <code>ng-template</code> containing menu items, using the standard menu item markup</li>
+        <li>attach your template to the parent menu item by passing the <code>TemplateRef</code> to its
+            <code>[subMenu]</code>
+        </li>
+    </ul>
+    The important thing is to create templates with submenu items inside the <code>fd-menu</code> tag, so Angular correctly
+    attaches them to the DI tree.
+</description>
 <component-example>
-    <fd-menu-separator-example></fd-menu-separator-example>
+    <fd-menu-with-submenu-example></fd-menu-with-submenu-example>
 </component-example>
-<code-example [exampleFiles]="menuSeparator"></code-example>
+<code-example [exampleFiles]="menuWithSubmenu"></code-example>
 
 <separator></separator>
 
-<fd-docs-section-title [id]="'keyboard'" [componentName]="'menu'">
-    Keyboard Support
+<fd-docs-section-title [id]="'mobile-menu'" [componentName]="'menu'">
+    Menu in mobile mode
 </fd-docs-section-title>
 <description>
-    Keyboard support can be added on every component which uses the <code>MenuComponent</code>. It simulates click event
-    on <code>Enter</code> or <code>Space</code> key down. It is also possible to override functions for a variety of
-    situations. For example, when the focus should escape the element. Use the up and down arrows to test the component
-    below. To enable focusing on non-focusable elements, change value of <code>tabIndex</code> to 0.
+    To open Menu in mobile mode use the <code>[mobile]="true"</code> option.
+    In mobile mode, the Menu opens inside the Dialog component and takes up the whole viewport of the device.
+    When going through nested levels of the Menu, the parent menu name will be displayed in the Dialog's header.
+    To add initial header visible in the Dialogs header at the first level on the Menu use
```

Comment:
To add an initial header in the first level of the Menu, use <code>9mobileMenuTitle]</code>